taxonomy/tasks.py

```diff
@@ -27,3 +27,19 @@ def update_course_skills(course_uuids):
         utils.refresh_course_skills(courses, should_commit_to_db=True)
     else:
         LOGGER.warning('[TAXONOMY] No course found with uuids [%d] to update skills.', course_uuids)
+
+
+@shared_task()
+def update_program_skills(program_uuids):
+    """
+    Task to update program skills.
+
+    Arguments:
+        program_uuids (list): uuids of programs for which skills needs to be updated
+    """
+    LOGGER.info('[TAXONOMY] refresh_program_skills task triggered')
```

Comment:
please add the program UUID in the logs.

Reply:
There will be logs for each program_uuid in `refresh_program_skills` so I don't think it is necessary to add them here too.